tests/deploy/test_scheduler_agent.py

```diff
@@ -871,7 +886,6 @@ async def test_deploy_event_propagation(agent: TestAgent, make_resource_minimal)
 
     def make_resources(
         *,
-        version: int,
```

Comment:
Oh wow, I've written so many tests like this, and I never realized that the version wasn't used...